src/main/java/com/apaliakou/Consequent.java

```diff
@@ -0,0 +1,176 @@
+package main.java.com.apaliakou;
+
+import java.util.SortedMap;
+
+class Data { // создал класс для исходных данных
+    void showmenu () { // создал метод отображения исходного меню
+        System.out.println(); // добавил пробел
+        System.out.println("МЕНЮ ПРОГРАММЫ, ОТВЕЧАЮЩЕЙ НА ЗАПРОС ПОЛЬЗОВАТЕЛЯ: ");
+        System.out.println("1. algorithmld - тип алгоритма;");
+        System.out.println("2. loopType - тип циклов, которые нужно использовать;");
+        System.out.println("3. n - параметр, передаваемый в алгоритм.\n");
+    }
+    void algorithmld (int firstparagraph) { // создал метод отображения информации пункта 1 исходного меню
+         switch (firstparagraph) { // добавил переключатель вариантов информации пункта 1 исходного меню
+            case 1: // вариант 1
+                System.out.println("1. вычисление ряда чисел Фибоначчи;");
+                break;
+            case 2: // вариант 2
+                System.out.println("2. вычисление факториала.");
+                break;
+        }
+    }
+    void loopType (int secondparagraph) { // создал метод отображения информации пункта 2 исходного меню
+        switch (secondparagraph) { // добавил переключатель вариантов информации пункта 2 исходного меню
+            case 1: // вариант 1
+                System.out.println("1. цикл while;");
+                break;
+            case 2: // вариант 2
+                System.out.println("2. цикл do-while;");
+                break;
+            case 3: // вариант 3
+                System.out.println("3. цикл for.");
+                break;
+        }
+    }
+}
+class Consequent { // создал класс для основных расчетов
+public static void main (String[] args) {
+    Data object = new Data (); // создал объект для задействования методов при расчетах (ссылок на методы)
+    object.showmenu(); // задействовал метод отображения исходного меню (через ссылку на метод)
+    for (int i = 0; i < args.length; i++) { // ввел счетчик для ввода значений аргументов через командную строку
```

Comment:
Be better if you check count of passed args[] here
if (args == null || args.length < 3) throw new Exception("Error in params quantity!");